cmd/juju/action/show.go

```diff
@@ -71,7 +74,11 @@ func (c *showCommand) Info() *cmd.Info {
 		Doc:     showActionDoc,
 	})
 	if featureflag.Enabled(feature.JujuV3) {
+		info.Name = "show-function"
```

Comment:
Ick...

Reply:
Yeah, but it's the same command, just with a different name. Achieves the result with very little code. So smaller amount of churn when flag is removed.